bob/src/api/http.rs

```diff
@@ -485,6 +494,34 @@ async fn read_directory_children(mut read_dir: ReadDir, name: String, path: Stri
     }
 }
 
+#[get("/data/<key>")]
+fn get_data(bob: State<BobServer>, key: BobKey) -> Result<Content<Vec<u8>>, StatusExt> {
+    let opts = BobOptions::new_get(None);
+    let result = runtime()
+        .block_on(async { bob.grinder().get(key, &opts).await })
+        .map_err(|err| -> StatusExt { err.into() })?;
```

Comment:
Are you sure that `?` doesn't perform conversion automatically? 
Because you have `impl From<Error> for StatusExt`
Here and in other places below

Reply:
Fixed in `http.rs`. Conversion doesn't perform in some places in `s3.rs`